distribution/kernel/carbon-home/bin/wso2server.sh

```diff
@@ -284,7 +284,8 @@ echo "Using Java memory options: $JVM_MEM_OPTS"
 #To monitor a Carbon server in remote JMX mode on linux host machines, set the below system property.
 #   -Djava.rmi.server.hostname="your.IP.goes.here"
 
-JAVA_VER_BASED_OPTS="--add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens java.rmi/sun.rmi.transport=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED"
+JAVA_VER_BASED_OPTS="--add-opens=java.base/java.net=ALL-UNNAMED --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens java.rmi/sun.rmi.transport=ALL-UNNAMED --add-opens=java.base/java.io=ALL-UNNAMED
+    --add-opens=java.xml/com.sun.org.apache.xerces.internal.dom=ALL-UNNAMED"
```

Comment:
Did you test this bat file in Windows?

Reply:
no. I don't have the windows env

Reply:
@senthalan  I took it to the next line because it exceeds 80 characters. I tested it on the latest pack, it worked without any issues. 

So shall i keep it or add it  the same line?

Reply:
@piraveena Tested the fix in windows, It resolves the warnings.